omg/mechanics/movement.py

```diff
@@ -21,6 +22,10 @@ def __init__(self, forward, backward, left, right):
         self.pressed_left: bool = False
         self.pressed_right: bool = False
 
+        # Track animations
+        self.move_direction = (0,0)
```

Comment:
Indeed, the `move_direction` should be part of this class and it should be propagated to the `Animations` class.